src/analyz/mod.rs

```diff
@@ -292,8 +273,14 @@ impl<'tcx> ToJson<'tcx> for mir::PlaceElem<'tcx> {
                     "from_end": from_end
                 })
             }
-            &mir::ProjectionElem::Subslice { ref from, ref to } => {
-                json!({"kind": "Subslice", "from": from, "to": to})
+            &mir::ProjectionElem::Subslice { ref from, ref to, from_end } => {
+                json!({
+                    "kind": "Subslice",
+                    "from": from,
+                    "to": to,
+                    // FIXME add to mir-verifier
+                    "from_end": from_end,
```

Comment:
Has this been added yet?

Reply:
It actually has - I'll remove the comment.